package.json

```diff
@@ -0,0 +1,43 @@
+{
+  "name": "book-management",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
+    "prop-types": "^15.7.2",
+    "react": "^16.12.0",
+    "react-dom": "^16.12.0",
+    "react-route-hook": "^1.2.1",
```

Comment:
we don't need some of these for this project. here are the requirements from pathwrite.

![AddReduxAndReact-Redux](https://user-images.githubusercontent.com/10922508/69742759-96dd8a80-113d-11ea-817a-5c1a659a45a0.png)